tests/printing/i13306.check

```diff
@@ -0,0 +1,14 @@
+[[syntax trees at end of                     typer]] // tests/printing/i13306.scala
+package example {
+  class MyClass() extends Object() {}
+  class MembersContainer() extends Object() {
+    type MyType[T >: Nothing <: example.MyClass] = Comparable[T]
+  }
+  final lazy module val Exports: example.Exports = new example.Exports()
+  final module class Exports() extends Object() { this: example.Exports.type =>
+    val instance: example.MembersContainer = new example.MembersContainer()
+    export example.Exports.instance.*
+    final type MyType[T <: example.MyClass] = Comparable[T]
```

Comment:
i noticed the lower bound does not print either, implying it's `Nothing`, however it is still inconsistent with how a `TypeBoundsTree` is printed - if you look at the original `MyType` - but that would lead to very verbose error messages I think.

One solution could be some print mode parameter which knows if we are printing "full trees" or just the type, which could allow you to decide when to print the lower bound